index.html

```diff
@@ -72,15 +72,14 @@ <h1>I’m Tracy<br />Glad to see,You!</h1>
         <div class="item1b">
           <h1>Tonic</h1>
           <div class="frame2">
-            <span>CANOPY</span>
+            <span style="font-size: 18px; font-weight: 500; color: #344563;">CANOPY</span>
             <ul class="role">
               <li>Back End Dev</li>
               <li>2015</li>
             </ul>
           </div>
           <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
           </p>
           <div class="tags">
             <ul>
```

Comment:
- [ ] Kindly include the languages used in your project cards and also implement the `hover`  functionality to your buttons to match the original design

![image](https://user-images.githubusercontent.com/98453979/213172197-ca3c40b0-a781-4e43-8899-16e7b48dcd7f.png)

- [ ] Kindly note that in your `about me` section, elements in the language cards are supposed to be aligned vertically and not horizontally

![image](https://user-images.githubusercontent.com/98453979/213173146-6a7b124f-8764-4251-8c19-a944f7771e56.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/213173458-0d18ac55-1c1a-49b7-95e3-a0cfcc9c8647.png)

- [ ] Kindly adjust the height of your inputs in the contact section to match the original design

![image](https://user-images.githubusercontent.com/98453979/213173853-81fe64a6-8c98-486e-80e0-907b7f0c28cf.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/213174252-51ff86af-71ad-4d60-942d-2f05642fda1b.png)

Reply:
[OPTIONAL] You are using `.sub-lang a` class for its styling. Kindly edit this span element according to the FIGMA design. According to Figma:
```
font-weight: 500;
font-size: 15px;
```
![image](https://user-images.githubusercontent.com/61361037/213233704-09c5b899-c61e-4d70-b75e-f7166a898464.png)

Your design looks like this:
![image](https://user-images.githubusercontent.com/61361037/213234160-6fac9be5-1444-42d7-9a50-52b3fd22c07e.png)